Add top navigation component

milolucero opened the pull request:
### Changes
- Adding reusable top navigation component
- Adding base images for the project
- Importing the project fonts globally

### Missing
- Styles on hover
- Responsive design

bandrson commented:
My only suggestion is that we change the underscores to hyphens in the asset file names. Everything else looks great!

milolucero commented:
> My only suggestion is that we change the underscores to hyphens in the asset file names. Everything else looks great!

I agree, let's handle multi-word filenames with hyphens from now on. I renamed the assets replacing the underscores with hyphens and adjusted the paths in the code accordingly.
